deployment/express/package.json

```diff
@@ -4,15 +4,12 @@
   "description": "",
   "main": "server.js",
   "dependencies": {
-    "compression": "^1.7.4",
     "express": "^4.17.1",
-    "http-proxy-middleware": "^1.0.5",
-    "path": "^0.12.7",
-    "yargs": "^15.4.1"
+    "path": "^0.12.7"
   },
   "devDependencies": {},
   "scripts": {
-    "start": "node server.js",
+    "start": "set PORT=9090 && set CONSUMER_API=https://opensource.bigusdatus.com/spline/rest/consumer&& node server.js",
```

Comment:
Please don't use `opensource.bigusdatus.com` as a default endpoint even for dev. That server isn't intended for public usage even though it is accessible. It's primarily for _our_ CI purposes and rare ad-hoc demos.
For dev convenience, I'd suggest to set it somewhere outside of the project codebase (env variable? global node config?).

Reply:
`set PORT=9090`
The readme says the default port is 7070, so as a user I'd expect doing `npm run start` with no additional arguments would imply those defaults.

Reply:
Oh. It should not be there. It is testing stuff. Will remove it. 

Ths.